.jshintrc

```diff
@@ -74,6 +74,7 @@
 		"console" : false,
 		"page"    : false,
 		"Polymer" : false,
-		"util"    : true
+		"util"    : true,
+		"Chart"    : true
```

Comment:
Honestly, i hoped that will get into master, so that one day @victorbalan  passes by and just dies of OCD